packages/ui/components/alert/Alert.tsx

```diff
@@ -2,13 +2,18 @@ import { CheckCircleIcon, ExclamationIcon, InformationCircleIcon, XCircleIcon }
 import classNames from "classnames";
 import { ReactNode } from "react";
 
+import { Icon } from "../icon";
+
 export interface AlertProps {
   title?: ReactNode;
+  // @TODO: Message should be children, more flexible?
   message?: ReactNode;
+  // @TODO: Provide action buttons so style is always the same.
   actions?: ReactNode;
```

Comment:
I like this - however, we had a convo about this in dialog and no one used it and preferred passing in actions themselves as children

Reply:
Yeah fair point. I think it's good if we still have it as a ReactNode, but I'm thinking about exporting a custom component `AlertButton` for example, which at least ensures the buttons are always styled the same. Because right now, everyone adds their own buttons in there, so it's guaranteed that stuff will never look the same.